addons/l10n_fi/data/account_tax_report_line.xml

```diff
@@ -0,0 +1,115 @@
+    </record>
+    <record id="account_financial_report_line_fi_sale14_302" model="account.tax.report.line">
+        <field name="name">14%:n vero</field>
+        <field name="code">SALE14_302</field>
+        <field name="tag_name">302</field>
+        <field name="sequence">2</field>
+        <field name="country_id" ref="base.fi"/>
+        <field name="parent_id" ref="account_financial_report_line_fi_1"/>
+    </record>
+    <record id="account_financial_report_line_fi_sale10_303" model="account.tax.report.line">
+        <field name="name">10%:n vero</field>
+          <field name="code">SALE10_303</field>
+        <field name="tag_name">303</field>
+        <field name="sequence">3</field>
+        <field name="country_id" ref="base.fi"/>
+        <field name="parent_id" ref="account_financial_report_line_fi_1"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_material_eu_305" model="account.tax.report.line">
+        <field name="name">Vero tavaraostoista muista EU-maista</field>
+        <field name="code">PURCHASE_MATERIAL_EU_305</field>
+        <field name="tag_name">305</field>
+        <field name="sequence">4</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_service_eu_306" model="account.tax.report.line">
+        <field name="name">Vero palveluostoista muista EU-maista</field>
+        <field name="code">PURCHASE_SERVICE_EU_306</field>
+        <field name="tag_name">306</field>
+        <field name="sequence">5</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_import_payable_340" model="account.tax.report.line">
+        <field name="name">Vero tavaroiden maahantuonneista EU:n ulkopuolelta</field>
+        <field name="code">IMPORT_PAYABLE_340</field>
+        <field name="tag_name">340</field>
+        <field name="sequence">6</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_construct_318" model="account.tax.report.line">
+        <field name="name">Vero rakentamispalvelun ja metalliromun ostoista (käännetty verovelvollisuus)</field>
+        <field name="code">PURCHASE_CONSTRUCT_318</field>
+        <field name="tag_name">318</field>
+        <field name="sequence">7</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_purchase_307" model="account.tax.report.line">
+        <field name="name">Verokauden vähennettävä vero</field>
+        <field name="code">PURCHASE_307</field>
+        <field name="tag_name">307</field>
+        <field name="sequence">8</field>
+        <field name="country_id" ref="base.fi"/>
+    </record>
+    <record id="account_financial_report_line_fi_total_tax_payable" model="account.tax.report.line">
```

Comment:
I would put that line at the top and set it as the parent of all lines used in the computation.